code/game/jobs/job/service.dm

```diff
@@ -18,6 +18,24 @@
 	outfit_type = /decl/hierarchy/outfit/job/service/bartender
 	alt_titles = list("Waiting Staff","Barkeep","Mixologist","Barista" = /decl/hierarchy/outfit/job/service/bartender/barista)
 
+/datum/job/barista
```

Comment:
There's no need for a barista job - that's job bloat. Check the bartender's alt title, barista is one of them and comes with its own outfit.

Reply:
I wanted to see about separating them for improved business opportunities, whenever that wagon comes around. It's cool though, I can still nix this if you'd like.

Reply:
Yeah if you could. It's a fluff job that isn't too mechanically different for bartender. The business system would generate its own job anyhow.